steps/pytorch/callbacks.py

```diff
@@ -52,6 +53,13 @@ def on_batch_begin(self, *args, **kwargs):
     def on_batch_end(self, *args, **kwargs):
         self.batch_id += 1
 
+    def get_validation_loss(self):
+        if self.epoch_id not in self.validation_loss.keys():
```

Comment:
@apyskir there is a really cool method for dicts called .get(key, fallback_value). for instance here it would be self.validation_loss.get(self.epoch_id, score_model(self.model, self.function, self.validation_dataget)

Reply:
@jakubczakon Yes, but this method does not update the dictionary. But if the key is not in dict.keys() we want to perform dict[key]=score_model...